projects/Directory.Packages.props

```diff
@@ -38,12 +38,12 @@
     <PackageVersion Include="System.Net.Http.Json" Version="8.0.1" />
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
     <GlobalPackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" />
```

Comment:
You really shouldn't need to reference this package at all. I would think you'd be able to remove this entire `ItemGroup`.

Reply:
Thanks for noticing!